### PR TITLE
fix: normalized custom dockerfile

### PR DESCRIPTION
**Pull request by floralatin**

### CODE REVIEW

File: normalizer/core.py [hunk 1]
```diff
@@ -404,7 +405,7 @@ def normalize(
             f'The folder "{work_path}" does not exist, can not normalize'
         )
 
-    dockerfile_path = work_path / 'Dockerfile'
+    dockerfile_path = work_path / dockerfile if dockerfile else work_path / 'Dockerfile'
```
Comment (nomagick) [inline normalizer/core.py:408]:
```suggestion
    dockerfile_path = (work_path / dockerfile) if dockerfile else (work_path / 'Dockerfile')
```

File: normalizer/core.py [hunk 2]
```diff
@@ -370,6 +370,7 @@ def normalize(
     env: Dict = {},
     build_env: Dict = {},
     dry_run: bool = False,
+    dockerfile: str = None,
```
Comment (nomagick) [inline normalizer/core.py:373]:
```suggestion
    dockerfile: str = '',
```

Comment (floralatin) [reply normalizer/core.py:373]: If we change `dockerfile: str = None`  ro `dockerfile: str = ''`.
We determine whether there is only by `dockerfile == ''`, It's so inelegant.

Comment (nomagick) [reply normalizer/core.py:373]: Isn't that `''` also false?
You specified dockerfile to be `str` then it should not be None.
Or else use `Optional[str]`

Comment (floralatin) [reply normalizer/core.py]: > Isn't that '' also false?

You are right: '' is also false.

**Review by nomagick — APPROVED** [review]

**Pull request merged**